ast/mock_visitor_test.go

```diff
@@ -0,0 +1,39 @@
+// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
There's a `go:generate` declaration in ast/visitor.go. This repository doesn't
use a YARPC-like generate.sh.